micro-ui/web/micro-ui-internals/packages/svg-components/src/svg/SetUpMicroplanSvg.js

```
@@ -0,0 +1,45 @@
+}) => {
+  return (
+    <svg
+      width={width}
+      height={height}
+      viewBox="0 -960 960 960"
+      className={className}
+      style={style}
+      onClick={onClick}
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)